pkg/internalapis/featurestates/v1alpha1/register.go

```diff
@@ -0,0 +1,19 @@
+// NOTE: Boilerplate only.  Ignore this file.
+
+// Package v1alpha1 contains API Schema definitions for the cns v1alpha1 API group
+// +k8s:deepcopy-gen=package,register
+// +groupName=cns.vmware.com
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
```

Comment:
It would be good if we keep them separate since this is auto-generated for each API.

In the https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/internalapis/cnsoperator/register.go#L40 we have set the same version for all APIs.

Reply:
So far we have not kept it separate.

Reply:
For Migration as well I kept it separate - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/apis/migration/v1alpha1/register.go

Reply:
@SandeepPissay I have addressed your review comment here - https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/721/commits/b0e7e06a8158ac78e29dad84f995ce888e0cc724

if the change looks good, I can squash all commits and start testing.

Reply:
Lets discuss this internally as even I have seen discrepancies in our API code organization and operator-sdk generated code.